drivers/video/Kconfig.mcux_csi

```diff
@@ -3,9 +3,8 @@
 # Copyright (c) 2019, Linaro Limited
 # SPDX-License-Identifier: Apache-2.0
 
-DT_COMPAT_NXP_IMX_CSI := nxp,imx-csi
-
 config VIDEO_MCUX_CSI
 	bool "NXP MCUX CMOS Sensor Interface (CSI) driver"
-	default $(dt_compat_enabled,$(DT_COMPAT_NXP_IMX_CSI))
+	default DTS_HAS_NXP_IMX_CSI_ENABLED
```

Comment:
How come `DTS_` and not `DT_` like we use in other devicetree APIs?

Reply:
No real reason, I changed it to `DT_`